.github/workflows/gh_pages.yml

```diff
@@ -0,0 +1,46 @@
+name: Generate Doxygen documentation
+
+on:
+  schedule:
+  - cron: 0 1 * * *
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: 'sycl'
+        path: repo
+    - name: Install deps
+      run: sudo apt-get install -y doxygen graphviz ssh
```

Comment:
This is bad practice ti install something on the test machine as part of test job
If you have some unresolved dependencies please please request installation by Tools team.